index.html

```diff
@@ -7,57 +7,215 @@
+                        </p>
+                        <ul>
+                            <li class="list1">HTML</li>
+                            <li class="list2">Bootstrap</li>
+                            <li class="list3">Ruby</li>
+                        </ul>
+                        <button type="button">See project</button>
+                    </article>
+                    <article class="card">
+                        <h5>Profesional Art Printing Data</h5>
+                            <p>A daily selection of privately personalized reads; 
+                                    no accounts or sign-ups required. Has been the industry's standard.
+                        </p>
+                        <ul>
+                            <li class="list1">HTML</li>
+                            <li class="list2">Bootstrap</li>
+                            <li class="list3">Ruby</li>
+                        </ul>
+                        <button type="button">See project</button>
+                    </article>
+                    <article class="card">
+                        <h5>Profesional Art Printing Data</h5>
+                            <p>A daily selection of privately personalized reads; 
+                                    no accounts or sign-ups required. Has been the industry's standard.
+                        </p>
+                        <ul>
+                            <li class="list1">HTML</li>
+                            <li class="list2">Bootstrap</li>
+                            <li class="list3">Ruby</li>
+                        </ul>
+                        <button type="button">See project</button>
+                    </article>
+                    <article class="card">
+                        <h5>Profesional Art Printing Data</h5>
+                            <p>A daily selection of privately personalized reads; 
+                                    no accounts or sign-ups required. Has been the industry's standard.
+                        </p>
+                        <ul>
+                            <li class="list1">HTML</li>
+                            <li class="list2">Bootstrap</li>
+                            <li class="list3">Ruby</li>
+                        </ul>
+                        <button type="button">See project</button>
+                    </article>
+            </section>
+            <!-- about section -->
+            <section class="about">
+                <h2>About me</h2>
+                    <p>Hi..!! I’m a web developer! I can help you build a product, feature or website. Look 
+                                through some of my work and experience! If you like what 
+                                you see and have a project you need coded,
+                                don't hesitate to contact me.
+                </p>
+                <button type="button">Get my resume</button>
```

Comment:
- [ ] You can do away with the `type` attribute in the `button` on this line, for the default `type` attribute for a button is still `button` to avoid unnecessary repetitions 😀.

Reply:
`type` removed.